### DISCUSSION TITLE
3.5.0

### DISCUSSION THREAD

**Discussion by Chris2011** [Chris2011/minifierbeans#103] [General]
## What's Changed
* Update packages versions by @yoancutillas in https://github.com/Chris2011/minifierbeans/pull/102

## New Contributors
* @yoancutillas made their first contribution in https://github.com/Chris2011/minifierbeans/pull/102

<hr /><em>This discussion was created from the release <a href='https://github.com/Chris2011/minifierbeans/releases/tag/3.5.0'>3.5.0</a>.</em>

**Comment by matthewslyman** [comment]
Just installed on NetBeans 21, and tried this out by right clicking a file and trying to minify it.
It didn't work at first: I had to figure out from the error message that I needed to separately download and install Node.
If convenient, please consider making error messages more clear & explicit when a user attempts to minify something but doesn't have a dependency installed.
Thank you for your work on this!

**Comment by Chris2011** [comment]
Hey @matthewslyman thx for your info. There are 2 hints that  node is required.
First in the readme there is the hint (node based). Yes it could be overseen and maybe it is not complete understandable. I can fix this by adding a more detailed info.

The second hint is when you open Options -> HTML -> Minification, on each panel you have a bottom infomration that NodeJS is required. Can be overseen, I understand.

For now I will just add the info into the readme. Please create a ticket with the error message so that I can react to it.

Maybe I can make my plugin to work as a dependency that you need the HTML package installed to make it more clear that you need node.
